targets/hal/TARGET_Atmel/TARGET_SAM_CortexM0P/TARGET_ADAFRUIT_CP_G18A/PeripheralPins.c

```diff
@@ -75,10 +75,10 @@ const PinMap PinMap_SERCOM_PAD[] = {
     {PA31, SERCOM1_PAD3, 3},
     {PB02, SERCOM5_PAD0, 3},
     {PB03, SERCOM5_PAD1, 3},
-    {PB08, SERCOM4_PAD0, 3},
-    {PB09, SERCOM4_PAD1, 3},
-    {PB10, SERCOM4_PAD2, 3},
-    {PB11, SERCOM4_PAD3, 3},
+    {PB08, SERCOM2_PAD0, 2},
```

Comment:
@sabas1080 I'm not sure these changes to PB08 - PB11 are correct. Can you confirm you meant to include these in your PR?

Reply:
(feel free to tell me why I'm wrong 😄 )

Reply:
when the two of them are in SERCOM4 there is conflict